internal/provider/common.go

```diff
@@ -134,11 +134,20 @@ func GetActualUpdateClusterPlatformParams(platform *models.Platform, userManaged
 	return platform, userManagedNetworking, nil
 }
 
-func GetActualCreateClusterPlatformParams(platform *models.Platform, userManagedNetworking *bool, highAvailabilityMode *string) (*models.Platform, *bool, error) {
+func GetActualCreateClusterPlatformParams(platform *models.Platform, userManagedNetworking *bool, highAvailabilityMode *string, cpuArchitecture string) (*models.Platform, *bool, error) {
 	if err := checkPlatformWrongParamsInput(platform, userManagedNetworking, nil); err != nil {
 		return nil, nil, err
 	}
 
+	if cpuArchitecture == models.ClusterCPUArchitectureS390x || cpuArchitecture == models.ClusterCPUArchitecturePpc64le {
+		if userManagedNetworking != nil && !*userManagedNetworking {
+			return nil, nil, common.NewApiError(http.StatusBadRequest, errors.Errorf("Can't disable user-managed-networking on %s architecture", cpuArchitecture))
```

Comment:
please use the same name as in the feature getName method
https://github.com/openshift/assisted-service/blob/master/internal/featuresupport/features.go#L262